packages/gatsby/src/components/hooks.tsx

```diff
@@ -0,0 +1,35 @@
+import {StaticQuery, graphql} from 'gatsby';
+import React                  from 'react';
+
+export function Hooks() {
+  return (
+    <StaticQuery
+      query={graphql`
+        query MyQuery {
+          yarnIntrospection(id: {eq: "0f47090c-afdf-5a02-98f8-047adfe6297f"}) {
+            value {
+              name
+              definition
+              comment
+              file
+            }
+          }
+        }
+      `}
+      render={data => <>
+        {data.yarnIntrospection.value.map(({name, comment, definition}) => (
+          <div key={name}>
+            <h3 id={`hook-${name}`}><code>{name}</code></h3>
+            {comment.split(/\n\n/).map((text, index) => (
+              <p key={index}>{text}</p>
```

Comment:
`text` should be rendered as markdown, otherwise backticks don't display as code.

Reply:
I don't know how to do this cleanly; Gatsby preprocesses the markdown at build time, but afaik there isn't a way to preprocess a json field this way.

Reply:
We can process the markdown at runtime just like we do for the JSON schema field descriptions:

```diff
diff --git a/packages/gatsby/src/components/hooks.tsx b/packages/gatsby/src/components/hooks.tsx
index 9b412e0c1..9c2604bbe 100644
--- a/packages/gatsby/src/components/hooks.tsx
+++ b/packages/gatsby/src/components/hooks.tsx
@@ -1,4 +1,5 @@
 import {StaticQuery, graphql} from 'gatsby';
+import marked                 from 'marked';
 import React                  from 'react';
 
 export function Hooks() {
@@ -21,7 +22,7 @@ export function Hooks() {
           <div key={name}>
             <h3 id={`hook-${name}`}><code>{name}</code></h3>
             {comment.split(/\n\n/).map((text, index) => (
-              <p key={index}>{text}</p>
+              <div key={index} dangerouslySetInnerHTML={{__html: marked(text)}} />
             ))}
             <pre><code>
               {definition}
```